FirebaseInAppMessaging.podspec

```diff
@@ -44,6 +45,7 @@ See more product details at https://firebase.google.com/products/in-app-messagin
   s.ios.dependency 'FirebaseAnalyticsInterop', '~> 1.3'
   s.dependency 'FirebaseInstanceID', '~> 4.0'
   s.dependency 'GoogleDataTransportCCTSupport', '~> 1.0'
+  s.dependency 'FirebaseABTesting', '~> 3.2.0'
```

Comment:
Firebase should allow minor version updates of Firebase deps. Change to '~> 3.2'